src/TBuffer.h

```diff
@@ -156,7 +156,7 @@ class TBuffer
     bool applyLink(const QPoint& P_begin, const QPoint& P_end, const QStringList& linkFunction, const QStringList& linkHist);
     bool applyFgColor(const QPoint&, const QPoint&, const QColor&);
     bool applyBgColor(const QPoint&, const QPoint&, const QColor&);
-    void appendBuffer(const TBuffer& chunk);
+    void appendBuffer(TBuffer &chunk);
```

Comment:
⚠️ This looks wrong - it means that you are changing the item supplied to the method as an argument - and that shouldn't be necessary/happening, please review your code!

... also our code formatting style (declared in `/src/.clang-format`) has the pointer/reference marks adjacent to the class and NOT the identifier as you are changing here.